Makefile

```diff
@@ -1,4 +1,6 @@
-URL := http://localhost:1313
+URL ?= http://localhost:1313
+PORT ?= 1313
```

Comment:
I would extract port from URL portion, unless there's any sense in allowing separate port than the one in URL?

Reply:
The URL is used in the generated HTML, whereas the port is used to bind; they can have different values.

Reply:
AFAIK if it has different values it won't work.

e.g running `make server PORT=1314` opens the browser on `http://localhost:1313`

![image](https://user-images.githubusercontent.com/51078027/157015266-ee4caa79-31a4-4cdb-8345-f155813a23f0.png)

If anything, specifying baseURL in this case is unnecessary since what in really needed is to specify bind IP to 0.0.0.0 (or whatever you want if you want it listening on just the one IP)

Reply:
I opened #696 to address this

Reply:
There are cases where one might want to bind to a port that’s not the same as that used in the generated HTML, _e.g._ if Hugo is run inside a container with the port exposed on a different port, or behind NAT.